requirements.txt

```diff
@@ -44,5 +44,6 @@ nbconvert==5.6.1
 pre-commit==2.13.0
 plotly==5.1.0
 dataframe-image==0.1.1
+h5netcdf==0.11.0
```

Comment:
my pycharm is indicating that `fiona` is imported in `arc2_precipitation.py` but not included in `requirements.txt`. Should it be included?

Reply:
I didn't put in there because it's a requirement of `geopandas` so should already be there (added because it allows excepting the specific errors from `gpd.read_file`), but good idea to be explicit (should be fine anyway when porting to toolbox).